src/lib/hooks/useOpenComposer.tsx

```diff
@@ -0,0 +1,21 @@
+import {useMemo} from 'react'
+import {Trans} from '@lingui/macro'
+
+import {useRequireEmailVerification} from '#/lib/hooks/useRequireEmailVerification'
+import {useOpenComposer as rootUseOpenComposer} from '#/state/shell/composer'
+
+export function useOpenComposer() {
+  const {openComposer} = rootUseOpenComposer()
+  const requireEmailVerification = useRequireEmailVerification()
+  return useMemo(() => {
+    return {
+      openComposer: requireEmailVerification(openComposer, {
+        instructions: [
+          <Trans key="pre-compose">
+            Before creating a post, you must first verify your email.
```

Comment:
```suggestion
            Before creating a post or replying, you must first verify your
            email.
```
nit: This message is shown when replying as well as creating a root post, so might be good to include that case too.